### PR TITLE
Material Weapon rework

### PR DESCRIPTION
**Pull request by Anewbe**
Most material weapons will go dull, rather than shatter. This removes the sharp and edge flags, and lessens the damage it does. Spears will still shatter.
- [x]  TODO: Add in whetstones
- [x] TODO: Make whetstones... Gettable?

### CODE REVIEW

**Comment by Serithi** [comment]
On-station blacksmith job making and fixing weapons when

**Comment by Anewbe** [comment]
Fixing weapons as soon as I get back around to it. Blacksmith probably neverish, go yell at an engineer.

**Comment by Serithi** [comment]
But muh medieval station

**Comment by Anewbe** [comment]
Engineering == Dwarves

Deal with it.

**Comment by Anewbe** [comment]
Found a bug with sandstone, can't actually craft anything special. Using plasteel instead, at least for now.

**Comment by Anewbe** [comment]
Only issue now is on the damage done, and whether or not we actually prefer this over the old system.

File: code/game/objects/items/weapons/material/twohanded.dm [hunk 1]
```diff
@@ -140,4 +141,5 @@
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	attack_verb = list("attacked", "poked", "jabbed", "torn", "gored")
 	default_material = "glass"
-	applies_material_colour = 0
\ No newline at end of file
+	applies_material_colour = 0
+	fragile = 1	//It's a haphazard thing of glass, wire, and steel
```
Comment (Yoshax) [inline code/game/objects/items/weapons/material/twohanded.dm]: Spear blades can be made out of almost any material, not just glass. You can apply steel blades, plasteel, diamond, uranium, etc.

Comment (Neerti) [inline code/game/objects/items/weapons/material/twohanded.dm]: I think it's not really the tip shattering but the rod itself.

**Pull request merged**